frontend/src/state/index.js

```diff
@@ -439,6 +552,7 @@ export function globalState() {
         // If the status is not 'canceled', just update the status
         ustate.openOrders[openOrderIndex].status.set(status);
       }
+      // update openOrdersPerTable
     },
 
     get restaurantId() {
```

Comment:
The code contains some disabled eslint rules, which means the program may have some potential bugs. Disabling eslint is not the ideal way of resolving issues, so it's recommended to fix the specific eslint errors instead.

The function `transformOpenOrdersToOpenOrdersPerTable` seems to be trying to group related open orders by table, but there are several issues with the implementation:

1. The function modifies the original input array rather than returning a new array, which can cause issues and unexpected outcomes.
2. There are some commented-out code blocks, which may suggest that the current implementation is unfinished or not fully tested.
3. The function has an O(n^3) time complexity due to nested loops, which can result in performance issues for large datasets.

It might be better to write a new function that creates a separate object to store open orders per table, which would avoid the above problems.

Other than that, the code seems okay at a glance.